eng/common/scripts/Verify-Resource-Ref.ps1

```diff
@@ -9,7 +9,7 @@ foreach ($file in $ymlfiles)
   $ymlContent = Get-Content $file.FullName -Raw
   $ymlObject = ConvertFrom-Yaml $ymlContent -Ordered
 
-  if ($ymlObject.Contains("resources"))
+  if (($null -ne $ymlObject) -and ($ymlObject.Contains("resources")))
```

Comment:
```suggestion
  if (!$ymlObject -and $ymlObject.Contains("resources"))
```

Reply:
Wouldn't it be the opposite?

```suggestion
  if ($ymlObject -and $ymlObject.Contains("resources"))
```

Reply:
Also this script does not have strict mode, so I think you could also do:

```suggestion
  if ($ymlObject?.Contains("resources"))
```

Reply:
Yes talk @benbp's suggestion :)

Reply:
I don't think @benbp suggestion works. At least from my local testing even after doing `Set-StrictMode -Off`

Reply:
That is a PS 7+ feature, so perhaps your PS version is older. Even with out that new feature his first suggestion is the correct one over mine.